### ISSUE TITLE
Would it be possible to optionally use jax.gaussian_kde instead of scipy?

### ISSUE THREAD

**Issue by NeilGirdhar** [mwaskom/seaborn#2884]
[Now that Jax has](https://github.com/google/jax/pull/11237) a GPU-accelerated version of `gaussian_kde`, would it be possible to use that instead of the scipy or the internal version of this function?

**Comment by mwaskom** [comment]
No, sorry, that would be too much increase in maintenance burden.
